pages.it/linux/adduser.md

```diff
@@ -1,7 +1,7 @@
 # adduser
 
 > Servizio per aggiungere utenti.
-> Maggiori informazioni: <https://manpages.debian.org/latest/adduser/adduser.html>.
+> Maggiori informazioni: <https://manned.org/adduser>.
```

Comment:
[Here](https://github.com/tldr-pages/tldr/pull/5738#discussion_r611495155), @patricedenis pointed out that `adduser`, `useradd`, `deluser` and `userdel` are  Debian specific.

Reply:
Hmm, that conversation seems to have been based more on suppositions than certainties. My doubts stem from the fact that https://manned.org/adduser lists versions of this page from (besides Debian & Ubuntu) CentOS, Fedora, and even FreeBSD. Maybe we should try to figure this out more definitively.

Reply:
I also didn't find much. What I found was that all four `manned.org` pages had

> Copyright (C) 1995 Ted Hajek, with a great deal borrowed from the original Debian adduser

on the bottom

Reply:
That seems to indicate that indeed the project exists independently in different distros. If we are to link to a man page, I'd rather link to manned.org where it's possible to access all the distros' manpages, than force users to the Debian one. What do you think?

Reply:
I think the main reason we should prefer official sources is that if a command has an official page that probably means, it's maintained and updated. Therefore the official documentation is (often) the most up to date. That isn't the case with very old commands like `adduser`, which (as far as I know) doesn't get updated. In that case any good source is probably as good as the official one.

Reply:
To conclude: I think `manned.org` is fine here

Reply:
@marchersimon  I think only `adduser` and `deluser` are Debian specific.

Reply:
Marking this one as resolved since @navarroaxel has approved the PR an the others have explicitly indicated support for using `manned.org`.